scripts/release/debian/deb_install.sh

```diff
@@ -83,6 +83,10 @@ setup() {
     fi
     echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" \
         > /etc/apt/sources.list.d/azure-cli.list
+    echo 'Package: azure-cli
+Pin: origin "packages.microsoft.com"
+Pin-Priority: 1001' \
+        > /etc/apt/preferences.d/azure-cli
```

Comment:
I don't think this works:

```
# echo 'Package: azure-cli
Pin: origin "packages.microsoft.com"
Pin-Priority: 1001' > /etc/apt/preferences.d/azure-cli

# apt-cache policy azure-cli
azure-cli:
  Installed: 2.30.0-2
  Candidate: 2.30.0-1~buster
  Version table:
 *** 2.30.0-2 500
        500 http://deb.debian.org/debian sid/main amd64 Packages
        100 /var/lib/dpkg/status
     2.30.0-1~buster 1001
        500 https://packages.microsoft.com/repos/azure-cli buster/main amd64 Packages
        500 https://packages.microsoft.com/repos/azure-cli buster/main all Packages
```

@bluca, what is the best way to prefer https://packages.microsoft.com/ regardless of the version?